view/theme/frio/css/image_grid.css

```diff
@@ -0,0 +1,25 @@
+* {
+	box-sizing: border-box;
+}
+
+.row {
```

Comment:
Can you confirm it doesn't conflict with the Bootstrap stylesheet we're using in Frio and might also make use of `class="row"`?

Reply:
see https://github.com/friendica/friendica/pull/12334#discussion_r1046413606

Reply:
Excellent.